### DISCUSSION TITLE
Home Office e Rotina

### DISCUSSION THREAD

**Discussion by manzoliric** [frontendbr/forum#1166] [Discussão]
Boa tarde pessoal, a pouco tempo começei a trabalhar como Home Office (2 meses) e estou me sentindo bem cansado, estou pensando em começar a fazer algum exercicio ou algo que me deixe menos cansado ao longo do dia.
Estou passando quase 20h sentado no computador e isso está me prejudicando bastante, dores nas costas, insonias e etc..

Alguem aqui trabalha Home Office e tem alguma rotina para ajudar nisso? como acordar cedo e correr ou planejar seu dia e etc.. ?

**Comment by kvnol** [comment]
Pomodoro é uma ótima.

**Comment by renanlara** [comment]
20h sentado no computador?
Cara, tenta reduzir essa carga horária. Ao meu ver, não vai ter exercício que resolva isso.

**Comment by manzoliric** [comment]
@kvnol sempre tento usar, as vezes funciona, as vezes não, quando me pego fazendo algo que é realmente complicado, nunca consigo parar em 30 minutos ou 15.

@renanlara Sim, estou pensando em sair um pouco, deitar, ver uma serie, andar... está sendo 10h trabalhando (com uma pausa de 1h) e 7h no mesmo computador só que estudando, jogando..

**Comment by TailoMateus** [comment]
@manzolirch Também trabalho home office, nesses últimos meses coloquei algumas coisas em prática para me tornar melhor.

Escrevi sobre: https://medium.com/@tailogonsalves/h%C3%A1-algum-tempo-mudei-minha-vida-e-voc%C3%AA-tamb%C3%A9m-pode-mudar-2a1ae521151e

Espero que te ajude :D

**Comment by nathpaiva** [comment]
mano 20h na frente do computador é algo que está bem errado! crie a rotina de trabalho / hora de almoço / hora de parar
Se não o que tu ganha tendo um trampo remoto está sendo jogado por agua abaixo! ;)


Trabalhei por 2 anos remoto e tinha uma rotina bem certa, mesmo que as vezes pudesse ocorrer de trabalhar um pouco mais eu sempre tentava manter o meu padrão e conseguia!

Acordava e tomava café da manhã e me arrumava.
Começava a trabalhar 9h
Parava para almoçar 13h e fazia uma pausa de 1h
e parava na maior parte das vezes 18h

Duas vezes na semana fazia exercício antes de começar a trabalhar e no resto da semana fazia exercício anoite ou aproveitava para fazer algo diferente! ;)

Colocava um limite de hora diário de produtividade no RescueTime e já era!

**Comment by matheusml** [comment]
@hugobessaa

**Comment by manzoliric** [comment]
@TailoMateus achei massa o conteudo, parabens, começei a praticar hoje mesmo e espero conseguir seguir.

@nathpaiva fiz isso tambem, acordei mais cedo, me alonguei, tomei café, sai com o cachorro e tudo sem olhar o celular / computador, para dar uma aliviada, ainda são 11h mas me sinto sem sono, logo vou dar uma pausa.

Obrigado aos comentarios, estão me ajudando bastante :)

**Comment by RatoX** [comment]
@manzolirch Cara vou contar um pouco da minha rotina já que mudei pra trampo 100% remoto esse ano.

Estou a 7 meses trabalhando puramente remoto, vim de alguns anos de trampo em escritórios "normais" e foi bem dificil me adaptar nos primeiros meses mas acredito que já achei meu fluxo de trabalho. No inicio eu achei que ia ser bem fodastico e ia sobrar tempo pra tudo, e na real sobra mas tu gasta todo com besteira caso não se organize.

Minha rotina, hoje se encontra assim:

06:30 Despertador
07:00 Banho matinal e emails
07:30 reunião do time
09:00 Tomar café (vendo meu feed do youtube) + levar cão pra correr + alguns exercicios de barra na pracinha 
10:30 trabaiar 
14:00 almoço
15:00 trabaiar
17:30 Academia || Adestramento do dog ( Eu alterno um dia tow adestrando o dog e no outro to malhando as perninhas pra ficar com um bumbum dahora )
18:30 casinha

O horário nortuno eu destino para coisas importantes, tipo **Zerar God of War no modo hard** e limpar a casa.


Essa rotina eu não consigo seguir todos os dias porém eu já tenho um plano diario, se eu não consegui fazer algo da lista eu **sei exatamente o que eu não fiz**

Demorou um tempo para eu me ajeitar mas eu sigo algumas filosofias minhas:

- O lugar de trabalho não pode ser o mesmo de descanso; Ou seja nada de joguinhos no PC **pra mim** por isso eu prefiro jogar no PS4 é bem mais fácil diferenciar o que é trampo e o que é zuera
- Eu não preciso fazer tudo da lista porém eu preciso saber o **que eu não fiz no dia**
- Tarefas domesticas são **obrigatórias** e devem ser tratadas como cartões no jira, tenho que ter espaço na agenda pra ela
- Carboidratos Pobres & Açucares me deixam preguiçoso, nada disso durante dias normais
- Tenho que andar ao menos 1km por dia pra aumentar o foco diario
- Tenho TDAH, em dias de muitas perda de foco recorro a medicação: preescrita e original; Nada de comprar pelo mercado livre
- Vitaminas em dias, exame de sangue trimestral e suplementação de vitaminas como Vitamina D e B ( quando essas duas estão em baixa eu me sinto extremamente desanimado )
- Estudar sempre, todos os meus sábados eu dedico ao menos 4 a 5 horas de estudo
- Sono é sagrado, 23:00 já estou dormindo e telefone no modo avião __se foda mundão__

**Comment by felipefialho** [comment]
Fala mano, tudo bem?

Trabalhei remoto por 2 anos, e de fato, a facilidade de você acordar e poder trabalhar sem nem precisar trocar de roupa, pode causar alguns problemas colaterais.

É importante cuidar deles.

- Pomodoro funciona MUITO bem remoto, como disse o @kvnol 
- Aproveite as facilidades da vida remoto e sai de casa, vai dar um role...
- Começa a fazer alguma atividade fisica que te faz bem, de repente, até no meio da tarde
- Evitar passar de 8h por dia na frente do computador

Trabalhar remoto tem suas vantagens, mas é muito importante criar hábitos saudáveis na rotina para não term problemas de saúde (mental e fisica) depois.

**Comment by hugooliveirad** [comment]
Estabelecer uma rotina ajuda bastante.

Eu, por exemplo, reservo boa parte da manhã para trabalhar. É quando eu me sinto mais produtivo. E acabo marcando coisas para fazer à tarde.

Para não passar o dia apenas programando, achei outras coisas que gosto de fazer. Vale um instrumento, cozinhar, exercícios,  relacionamentos... qualquer coisa que não envolve olhar pra uma tela.

Ainda assim, alguns dias passam batidos e percebo que passei o dia inteiro no computador.

Eu funciono bem quando tenho pressão externa. Prometer que cozinharei uma refeição, pagar por uma aula de treinamento funcional em grupo, fazer aulas de teclado em uma escola de música. Esses compromissos me impedem de ficar tempo demais na frente da tela.

**Comment by ninetails** [comment]
Bom, não tenho experiência com trabalho remoto, mas diria que se nada funcionar e vc morar em alguma metrópole (bem específico rs) repasso uma dica de um amigo meu que é pagar um espaço num coworking. Geralmente tem café incluso, nem sempre são as mesmas pessoas todos os dias, faz um networking, não suja nada em casa, a maioria tem hora pra fechar (não te forçando ficar madrugada acordado) :D

**Comment by galvaodev** [comment]
Já trabalhei como Home (7 mêses) isso fulltime, porem foi uma experiência muito boa, acredito que não seja pra qualquer pessoal, e aqueles que acham que e a maior paz e tranquilidade está enganado (rsrs), nesse tempo aprendi que nem sempre e bom ter a casa como local de trabalho fulltime, ali você acumula todo estresse diário e acaba levando pro seu conforto da noite (que no caso e o mesmo lugar). 

Mudei um pouco do meu dia a dia como:

- Acorda mais cedo para tomar um belo café.
- Tomar um banho para da aquela dispertada
- Fazer uma call e para rever todos os jobs do dia
- Almoçar (1hr a 1:30hr)
- finalizar o job no final da tarde
- Academia
- E ver netflix com a esposa

Bom, sem que nem sempre e mil maravilha assim pra muitos, pois a renda no final do mês bate com as contas, muitas vezes no inicio eu esticava e dormia só depois das 4hr da manhã, não indico pra ninguém, misturar o Home com Freela, você acaba se tornando dependente do trabalho, isso não e bom.

A dica e simples (seja home ou freela), tenha uma quantidade de horas exata pra executar ambos serviços, não somos máquinas então curta também sua vida e familia.
